recipe/meta.yaml

```diff
@@ -10,18 +11,24 @@ source:
   sha256: a84f4bb7c5da1fc396ac3ebf493fa470a2cf208924b4bd9c6321d65d4906df9e
 
 build:
+  number: 1
   noarch: python
-  number: 0
   script: {{ PYTHON }} -m pip install . --no-deps -vv
 
 requirements:
   host:
-    - python >=3.5
+    - python {{ python_min }}
     - pip
+    - setuptools
```

Comment:
Adds `setuptools` to `requirements/host` as it is [needed to build `pynvml`]( https://github.com/gpuopenanalytics/pynvml/blob/11.5.3/setup.py#L1 ). This was also suggested by the linter above ( https://github.com/conda-forge/pynvml-feedstock/pull/17#issuecomment-2513231234 ) and will soon be required